torchreid/optim/optimizer.py

```diff
@@ -143,10 +152,14 @@ def _build_optim(model,
             else:
                 decay.append(param)
 
-        assert len(list(model.parameters())) == len(decay) + len(bias_no_decay) + len(weight_no_decay)
```

Comment:
How it worked in case some params are have required_graph == false?

Reply:
required_grad?

Reply:
*requires_grad

Reply:
As usual, I guess. The optimizer will not update those parameters with grad==False. I guess it doesn't matter if we pass all parameters to the optimizer together or separately by chanks

Reply:
I mean if one of the parameters will be `requires_grad==False` this assert will be activated